tasks/api.py

```
@@ -28,21 +28,29 @@
+    auth = request.headers.get("authorization", "")
+    token = auth[7:].strip() if auth.startswith("Bearer ") else request.headers.get("x-api-key", "").strip()
+    if token:
         try:
-            import importlib
-            mod = importlib.import_module(mod_name)
-            return await mod.get_current_user(request)
-        except ModuleNotFoundError:
-            continue
-    from fastapi import HTTPException
+            from tokens import verify_token
+            payload = verify_token(token, token_type="access")
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)